Makefile

```diff
@@ -154,19 +154,30 @@ deploy: manifests tools ## Deploy controller in the configured Kubernetes cluste
 	kustomize build config/default | kubectl apply -f -
 
 manifests: ## Generate manifests e.g. CRD, RBAC etc.
-	$(MAKE) generate-manifests CONTROLLER_GEN_SRC=./apis/...
-	$(MAKE) -C apis/cli generate-manifests CONTROLLER_GEN_SRC=./...
-	$(MAKE) -C apis/config generate-manifests CONTROLLER_GEN_SRC=./...
+	$(MAKE) generate-manifests
+	$(MAKE) -C apis/addonconfigs generate-manifests
+	$(MAKE) -C apis/cli generate-manifests
+	$(MAKE) -C apis/config generate-manifests
+	$(MAKE) -C apis/core generate-manifests
+	$(MAKE) -C apis/run generate-manifests
 	$(MAKE) -C cli/runtime generate-manifests
 
 generate-go: $(COUNTERFEITER) ## Generate code via go generate.
-	PATH=$(abspath hack/tools/bin):"$(PATH)" go generate ./...
+	@for i in $(GO_MODULES); do \
+		echo "-- Running go generate ./... $$i --"; \
+		pushd $${i}; \
+		PATH=$(abspath hack/tools/bin):"$(PATH)" go generate ./...; \
```

Comment:
nit: to be consistent with the other generate targets, should we convert this to invoke a generate-go target in each of the Makefile as well?

Reply:
I tried with a make target for this in common.mk but couldn't get it to work so leaving this with fmt added.

Reply:
sgtm. we can address this as a followup.

Reply:
> but just atm the "make fmt" needs fixing as well
> https://github.com/vmware-tanzu/tanzu-framework/actions/runs/3222939850/jobs/5272513892#step:13:348

Maybe not, looks like the missing fmt target lines are harmless, because for now every module that needs to generate fakes also has a fmt target to go with it? Will monitor the verify check to see if this change kept the expected behavior.